### PR TITLE
Adds Contact menu to the side bar.

### PR DESCRIPTION
**Pull request by steelstyle**
Icomoon is the deprecated way to add icon therefore installed @styled-icons/boostrap.
Additionally update the MenuLink internal component to allow custom dom tree.

Resolves CaptainFact/captain-fact#111.

### CODE REVIEW

File: app/components/App/Sidebar.jsx [hunk 1]
```diff
@@ -39,18 +41,18 @@ export default class Sidebar extends React.PureComponent {
...
         className={classes}
         activeClassName="is-active"
         onClick={this.closeSideBarIfMobile}
-        title={children}
+        title={title ? title : children}
         {...props}
       >
         {iconName && <RawIcon name={iconName} />}
-        <span>{children}</span>
+        {customLink ? <React.Fragment>{children}</React.Fragment> : <span>{children}</span>}
```
Comment (Betree) [inline app/components/App/Sidebar.jsx:55]: I think we can avoid the fragment here

```suggestion
        {customLink ? children : <span>{children}</span>}
```

**Review by Betree — COMMENTED** [review]
Looks good to me, could you rebase on `staging` to solve the conflick on `pakcage-lock.json`?

File: app/components/App/Sidebar.jsx (hunk 1 above)
Comment (steelstyle) [reply app/components/App/Sidebar.jsx]: Fair point.

**Comment by steelstyle** [comment]
Updated.

**Review by Betree — APPROVED** [review]
Perfect, thank you!

**Pull request merged**